bindings/python/src/OpenSpaceToolkitPhysicsPy/Unit.cpp

```diff
@@ -38,10 +38,10 @@ inline void OpenSpaceToolkitPhysicsPy_Unit(pybind11::module& aModule)
             )doc"
         )
         .value(
-            "None",
-            Unit::Type::None,
+            "Dimensionless",
```

Comment:
This is great, but wouldn't you have to update the CPP as well?

Reply:
Yeah, I think `None_` is a fine solution, it's a small enough shift between python and cpp.